pages/linux/eyewitness.md

```diff
@@ -0,0 +1,24 @@
+# EyeWitness
+
+> Python based tool to gather screenshots of websites, as well as additional information such as header info and default credentials if possible.
+> More information: <https://www.christophertruncer.com/eyewitness-triage-tool>.
+
+- Scan a single website using `--single` flag:
+
+`EyeWitness --single {{website}}`
```

Comment:
```suggestion
`EyeWitness --single "{{website_url}}"`
```

Option fix: possible missing quotes: quotes can be added if the placeholder can contain some special characters for POSIX-like shells.